web/src/util.ts

```diff
@@ -94,7 +94,10 @@ export const same_recipient = function util_same_recipient(a?: Recipient, b?: Re
     return false;
 };
 
-export const same_sender = function util_same_sender(a: RawMessage, b: RawMessage): boolean {
+export const same_sender = function util_same_sender(
+    a: RawMessage | undefined,
+    b: RawMessage | undefined,
+): boolean {
     return (
         a !== undefined &&
         b !== undefined &&
```

Comment:
This function should be comparing sender user IDs, right? Seems like a relic.

Reply:
Also the only caller is in `message_ilst_view`, so maybe it makes more sense to inline this there as just `a.sender_id === b.sender_id`, after the `undefined` check?

Reply:
Yeah function dates to 95309539bc50d0a75cd141290309cf8e32cb466a, which predates having `sender_id` in the API.

Reply:
Great catch, doing a followup for this